docs/getting_started.md

```diff
@@ -132,36 +132,53 @@ Experiment with the graph range parameters and other settings.
 Let us make this more interesting and start some example targets for Prometheus
 to scrape.
 
-The Go client library includes an example which exports fictional RPC latencies
-for three services with different latency distributions.
+The next steps will depend what operating system you're running. You only need to
+follow one of them, and after that you can move on to the next section. By the end
+of the next sub-sections you should have example targets listening on
+http://localhost:8080/metrics, http://localhost:8081/metrics, and
+http://localhost:8082/metrics.
 
-Ensure you have the [Go compiler installed](https://golang.org/doc/install) and
-have a [working Go build environment](https://golang.org/doc/code.html) (with
-correct `GOPATH`) set up.
+### Setup node_exporter on Linux/Mac/Unix
```

Comment:
```suggestion
### Set up node_exporter on Linux/Mac/Unix
```

Reply:
setup = noun
set up = verb

Reply:
Ended up needing to remove section header :(